doc/man7/provider-keymgmt.pod

```diff
@@ -309,6 +305,10 @@ See also L<provider-keyexch(7)> for the related
 B<OSSL_EXCHANGE_PARAM_EC_ECDH_COFACTOR_MODE> parameter that can be set on a
 per-operation basis.
 
+=item "use-cofactor-flag" (B<OSSL_PKEY_PARAM_USE_COFACTOR_ECDH>) <integer>
+
+Alias for B<OSSL_PKEY_PARAM_USE_COFACTOR_FLAG>.
```

Comment:
I would have just one item, and present both macros in it. It's a bit confusing having two items with the exact same name string.

Reply:
Maybe I need to remind that the name strings are governing. The macros are really just there for our convenience.

Reply:
What about b9af19db64b4a135987646415ef6ae112846e7c9 ?

Is the `=item` line too long? If so how should I split it?

Reply:
A line break would do well. It won't affect the output, the POD translators treat the whole file in paragraphs, they read and treat chunks separated by empty lines.

Reply:
... with exception for the verbatim (indented) stuff. That's the only place where line breaks are kept.

Reply:
Please see d69e24e4fc

The github rendering does not look too nice with the line break: https://github.com/openssl/openssl/blob/d69e24e4fc710128b6542d98fc44f9cc2bf17044/doc/man7/provider-keymgmt.pod